intro_to_ggraph.Rmd

```diff
@@ -0,0 +1,213 @@
+# A brief introduction to ggraph
+
+Ying Bi and Gaoge Liu
+
+```{r , include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggraph)
+library(igraph)
+library(dplyr)
+library(tidygraph)
+library(tidyr)
+library(readr)
+nodes <- read_csv("dolphin_nodes.csv")
+edges <- read_csv("dolphin_edges.csv")
```

Comment:
One last question before I merge your branch into our main bookdown, are these two csv files uploaded to GitHub as well? If not it might cause a problem in rendering the book.

I see in line 123 and 124 nodes and edges are read in again, so are these two lines of code (13 and 14) repetitive and can be deleted?

Reply:
Yeah, you are right. I have deleted lines 13 and 14 and it may be ok to merge now.